devel/libanalysis/include/ert/analysis/analysis_module.h

```diff
@@ -112,6 +112,7 @@ typedef enum {
   const char           * analysis_module_get_table_name( const analysis_module_type * module);
   const char           * analysis_module_get_name( const analysis_module_type * module );
   bool                   analysis_module_check_option( const analysis_module_type * module , long flag);
+  void                   analysis_module_set_module_data_block_vector( analysis_module_type * module, void* arg );
```

Comment:
No - we should *not* set a module data object on the `fwd_step` structure - that should just be passed as an argument to the updateA routine.